### PR TITLE
LPS-145185 SF

### PR DESCRIPTION
**Pull request by kevenleone**

### CODE REVIEW

**Comment by kevenleone** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
@bryceosterhaus I don't know where this SF error is coming from

I already ran: `gw formatSource` and `ant format-source-current-branch` and it's not catching this up

File: modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/layout/layout-screen/AddNewTabButton.d.ts [hunk 1]
```diff
@@ -12,7 +12,5 @@
  * details.
  */
 
-/// <reference types="react" />
```
Comment (jairmedeiros) [inline modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/layout/layout-screen/AddNewTabButton.d.ts:15]: @kevenleone, I think that the error comes from this line, try to added just for testing

**Comment by bryceosterhaus** [comment]
Updated in my original PR, just waiting on CI relevant to clear.

https://github.com/liferay-frontend/liferay-portal/pull/1816

**Pull request closed without merging**